### PR TITLE
docs: Doc to provide a script to watch the cou logs

### PR DESCRIPTION
**Pull request by jneo8**

### CODE REVIEW

File: docs/how-to/watch-cou-logs.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Below is the script to watch the COU logs when doing the development.
+This script watches every log file in the directory, even newly created files.
+
+.. code:: bash
+
+    #!/bin/bash
+
+    # Change LOG_DIR to the target directory
+    DIRECTORY="$LOG_DIR"
```
Comment (samuelallan72) [inline docs/how-to/watch-cou-logs.rst:13]: Isn't this a standard location?  Can we hardcode this to `~/.local/share/cou/logs/*`, so by default users who copy/paste the script don't need to make changes?

**Review by samuelallan72 — CHANGES_REQUESTED** [review]
Thanks, looks good!  Just a comment about the var that needs changing manually.

File: docs/how-to/watch-cou-logs.rst (hunk 1 above)
Comment (jneo8) [reply docs/how-to/watch-cou-logs.rst]: Updated

File: docs/how-to/watch-cou-logs.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Below is the script to watch the COU logs when doing the development.
+This script watches every log file in the directory, even newly created files.
+
+.. code:: bash
+
+    #!/bin/bash
+
+    # Change LOG_DIR to the target directory
+    DIRECTORY="~/.local/share/cou/log/"
```
Comment (samuelallan72) [inline docs/how-to/watch-cou-logs.rst:13]: I tried this, and it didn't work with the tilde home shortcut unfortunately.

```suggestion
    DIRECTORY="$HOME/.local/share/cou/log/"
```

File: docs/how-to/watch-cou-logs.rst (hunk 1 above)
Comment (samuelallan72) [reply docs/how-to/watch-cou-logs.rst]: Ah sorry I realised the path I suggested wasn't correct, but you got it right, thanks.

File: docs/how-to/watch-cou-logs.rst (hunk 2 above)
Comment (jneo8) [reply docs/how-to/watch-cou-logs.rst]: Updated

**Review by samuelallan72 — APPROVED** [review]

**Review by chanchiwai-ray — APPROVED** [review]

**Pull request merged**